provision/testdata/base_case.kube.yaml

```diff
@@ -59,6 +59,7 @@ data:
                 "start": "10.2.0.2"
             }
         ],
+        "pod-subnet-chunk-size": 32,
```

Comment:
We should ideally be testing, setting this to a non-default value, and in another case where nothing is set, and the default gets picked up.

Reply:
Testing a value set externally in testdata/with_overrides.kube.yaml